README: Header redesign

liamcharger opened the pull request:
This PR enhances the visual appeal of the main README.md header with a restyled header design. Open to feedback and suggestions for further improvements!

Additionally, in the README body, I've included a reference to [InfiniTimeExplorer](https://infinitimeexplorer.netlify.app/) since its existence is not widely known.

Please refer to [my InfiniTime fork](https://github.com/liamcharger/InfiniTime) for a preview of the changes.

rambonette commented:
Looks very nice imo 👏

NeroBurner reviewed (approved):
this looks great! I didn't know you could combine html tags with markdown

I especially like the collage. I'm curious, how did you create it?

liamcharger commented:
> this looks great! I didn't know you could combine html tags with markdown
> 
> 
> 
> I especially like the collage. I'm curious, how did you create it?

Glad you like it! I did it in Adobe Photoshop by taking several screenshots from InfiniSim ;) and then merging each screen with a representation of the physical watch that was used in InfiniLink. Then I positioned everything, added the screen that's stacked on the others, and lastly, I added a circle with a Gaussian blur behind everything for the shadow effect.

tituscmd commented:
This is beautiful. Great work!

mark9064 commented on README.md:
```diff
@@ -22,8 +38,13 @@ Fast open-source firmware for the [PineTime smartwatch](https://pine64.org/devic
+<br>
 
-***Note**: We removed mentions to NRFConnect as this app is closed source and recent versions do not work anymore with InfiniTime (the last version known to work is 4.24.3). If you used NRFConnect in the past, we recommend you switch to [Gadgetbridge](https://gadgetbridge.org/).*
+> *InfiniTimeExplorer is only compatible with web browsers that support Web BLE. Currently supported browsers include Chrome, Edge, and Firefox.* 
```
I could be misreading https://developer.mozilla.org/en-US/docs/Web/API/Web_Bluetooth_API, but it looks like firefox is not supported

mark9064 reviewed (commented):
Looks really pretty :)

FintasticMan commented:
I like the way it looks, but I do find it a bit strange that the yellow in the picture is darker than it "should" be.

liamcharger commented:
> I do find it a bit strange that the yellow in the picture is darker than it "should" be.

Not sure if it really matters, but do you mean the yellow in the InfiniTime logo?

FintasticMan commented:
Yeah, the yellow in the InfiniTime logo is darker than other places we use it. It looks like the colours are inverted actually, so I can see how it happened. I feel like maybe you could use the picture we use for the recovery firmware, that's on a black background.

FintasticMan commented:
@liamcharger I think you accidentally pushed an unrelated change to this branch. We recommend using separate branches for each PR, and not using main, to prevent this happening by accident.

liamcharger commented:
Oops, that was an accident...thought I was on a different branch.

liamcharger commented:
> I feel like maybe you could use the picture we use for the recovery firmware, that's on a black background.

@FintasticMan do you know where I can get an image resource of the recovery firmware picture?

mark9064 reviewed (commented):
Compare the colour of the centre picture to what's currently on the README
![image](https://github.com/user-attachments/assets/d2d9e885-5257-4008-aa44-a0a2d6971479)

I'm assuming Fin means this?

liamcharger commented:
Ok, I'll do that.

liamcharger commented:
@FintasticMan @mark9064 @JF002 all done!

FintasticMan reviewed (approved):
Looks good to me! If you update it to the latest main I'll merge it.

liamcharger commented:
Great! The branch is already up to date, though.

FintasticMan commented:
Ah, sorry, I had rebase merge selected rather than squash merge.
